### PR TITLE
New topic on publishing cookbooks to multiple Chef servers

### PR DESCRIPTION
**Pull request by davidwrede**
Signed-off-by: David Wrede <dwrede@chef.io>

### CODE REVIEW

File: chef_master/source/publish_cookbooks_multiple_servers.rst [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+The ``delivery-sugar`` cookbook exposes some libraries and a resource that you can use to
+publish a cookbook (or multiple cookbooks) to multiple Chef servers or organizations. The following examples show how to publish to both a single Chef server and how to extend that methodology to publish to multiple Chef servers.
+
+Prerequisites 
+===============================================================
+
+Before you begin, you must copy the ``knife.rb`` file 
+and the client key .pem file (referenced as ``client_key`` in ``knife.rb``) to the build nodes/runners that you will use in the build-cookbook. This can 
+be done manually by logging in to the build nodes/runners and copying the files.
```
Comment (afiune) [inline chef_master/source/publish_cookbooks_multiple_servers.rst]: I would probably mention that this process could be automated.. But I guess it is not very important for this topic, at the end of the day users will try to automated so it is up to you to add it or not, I'm fine either way.

File: chef_master/source/publish_cookbooks_multiple_servers.rst [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+To publish a cookbook to a Chef server, use the ``delivery_chef_cookbook`` resource and reference the ``knife.rb`` file that you copied to your build node/runner.
+
+This example shows how to publish a cookbook called ``rally`` to a single Chef server. 
+
+.. code-block:: ruby
+
+   knife_rb = 'path/to/the/knife_rb/file/in/the/build-node/knife.rb'
+
+   delivery_chef_cookbook 'rally' do
+     path 'path/to/the/cookbook/in/the/build-node/rally'
```
Comment (afiune) [inline chef_master/source/publish_cookbooks_multiple_servers.rst]: I would like to start the path with `/` so they know it is an absolute path. Relative could also work but they might need to know where are they located on the phase run which would be a bit tricky.

File: chef_master/source/publish_cookbooks_multiple_servers.rst [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+For this particular example, you want the cookbook uploaded at the very end of the workflow pipeline, in the **Functional** phase of the **Delivered** Stage.
+This requires that you modify the **Functional** recipe (``recipes/functional.rb``) of the ``build-cookbook`` within your project, as shown below.
+
+.. code-block:: ruby
+
+   # Run it only in Delivered::Functional
+   #
+   # This helper is coming from delivery-sugar
+   # => https://github.com/chef-cookbooks/delivery-sugar/blob/master/libraries/delivery_dsl.rb#L105,L113
```
Comment (afiune) [inline chef_master/source/publish_cookbooks_multiple_servers.rst:69]: awesome! 👏

**Comment by afiune** [comment]
A few nit's but overall LGTM. Would be good to mention (as we talked) that the default action for the resource is `:upload`..

**Comment by davidwrede** [comment]
gif-keyboard-5075811498912401752

**Pull request merged**